waimak-core/src/main/scala/org/apache/spark/ui/WaimakExecutionEvent.scala

```diff
@@ -0,0 +1,22 @@
+package org.apache.spark.ui
+
+import com.coxautodata.waimak.dataflow.spark.ui.{WaimakEventListener, WaimakGraph}
+import org.apache.spark.SparkContext
+import org.apache.spark.scheduler.SparkListenerEvent
+
+case class WaimakExecutionEvent(executionId: String, actionDescriptions: Seq[String], flowGraph: WaimakGraph) extends SparkListenerEvent
```

Comment:
I don't think we need the actionDescriptions in here anymore as that was to build the table (which we don't need)